ilc/server/tailor/merge-configs.js

```diff
@@ -0,0 +1,15 @@
+const deepmerge = require('deepmerge');
+
+module.exports = (original, override) => {
```

Comment:
You need to do a deep clone of the original object before it's modification. As same object may be shared between requests.

Reply:
I planed to do it, but I don't see any case to reproduce it, so decided to speed up with don't do cloning. but you are right, it's better to do it, it's more safe